locales/en/services.json

```diff
@@ -0,0 +1,35 @@
+[
+    {
+        "servicesgroup.cdn.name": "Cdn"
+    },
+    {
+        "servicesgroup.dating.name": "Dating"
+    },
+    {
+        "servicesgroup.gambling.name": "Gambling"
+    },
+    {
+        "servicesgroup.gaming.name": "Gaming"
+    },
+    {
+        "servicesgroup.hosting.name": "Hosting"
+    },
+    {
+        "servicesgroup.messenger.name": "Messenger"
+    },
+    {
+        "servicesgroup.privacy.name": "Privacy"
+    },
+    {
+        "servicesgroup.shopping.name": "Shopping"
+    },
+    {
+        "servicesgroup.social_network.name": "Social_network"
+    },
+    {
+        "servicesgroup.software.name": "Software"
```

Comment:
too generic name, better to make it more specific, e.g. Software development, dunno